packages/ember-cli-fastboot/test/request-details-test.js

```diff
@@ -32,7 +33,7 @@ describe('request details', function () {
       .create('request', {
         skipNpm: true,
         emberVersion: 'latest',
-        emberDataVersion: 'latest',
+        emberDataVersion: '~3.19.0',
```

Comment:
do we have problems with ember-fetch + ember-data?

Reply:
https://github.com/ember-cli/ember-fetch/pull/633

Reply:
I forgot if this is related to the error I saw in between commits. I'm just pinning ember-data to the version used in the project.
The ember-fetch issue is real, also seen at https://github.com/embroider-build/embroider/blob/28460bf58e75e653d917c2a97b6e616abf02d75b/tests/scenarios/fastboot-app-test.ts#L22 but I do not know why

Is the ember-fetch linked issue related?